### PR TITLE
assign default encryption scheme for passphrase with none.

### PR DESCRIPTION
**Pull request by KtorZ**
# Issue Number

#1701 

# Overview

- a4dd767006047e0fc14fb6de7b20920ae854dc03
  :round_pushpin: **assign default encryption scheme for passphrase with none.**

# Comments

cc @nikolaglumac

### CODE REVIEW

**Review by paweljakubas — APPROVED** [review]
looks like a fix that addresses the case when we had updated password that had not passphrase schema  and was not exposed (because resulted in Nothing)   

Alternatively, we could solve this via manual migration...

File: lib/core/src/Cardano/Wallet/DB/Sqlite.hs [hunk 1]
```diff
@@ -378,6 +414,7 @@ migrateManually tr defaultFieldValues =
                 _ <- Sqlite.step addColumn
                 Sqlite.finalize addColumn
       where
+        activeSlotCoeff = DBField CheckpointActiveSlotCoeff
```
Comment (KtorZ) [inline lib/core/src/Cardano/Wallet/DB/Sqlite.hs:417]: moved from above.

File: lib/core/src/Cardano/Wallet/DB/Sqlite.hs [hunk 2]
```diff
@@ -353,6 +388,7 @@ migrateManually tr defaultFieldValues =
                 Sqlite.finalize stmt
       where
         hardLowerBound = toText $ fromEnum $ minBound @(Index 'Hardened _)
+        rndAccountIx   = DBField RndStateAddressAccountIndex
```
Comment (KtorZ) [inline lib/core/src/Cardano/Wallet/DB/Sqlite.hs:391]: moved from above.

**Comment by KtorZ** [comment]
bors retry

**Pull request merged**